controlpanel/frontend/jinja2/webapp-admin-list.html

```diff
@@ -12,12 +12,17 @@ <h1 class="govuk-heading-xl">{{ page_title }}</h1>
   {% if request.user.has_perm('api.list_app') %}
   {{ app_list(apps, request.user) }}
 
-    {% if request.user.has_perm('api.create_app') %}
-    <p class="govuk-body">
-      <a class="govuk-button" href="{{ url('create-app') }}">
-        Register an app
-      </a>
-    </p>
+    {% if request.user.has_perm('api.is_superuser') %}
```

Comment:
I wonder if we should have this at the top of the page, as it could be easy to miss by having it at the bottom as it is such a long list in prod?

Reply:
Another alternative would be to do this in the Django Admin, implemented as an [admin action](https://docs.djangoproject.com/en/5.1/ref/contrib/admin/actions/). The benefit here would be that you can then use some of the built in django admin functionality in order to only export a subset of app admins, rather than all apps.